package.json

```diff
@@ -15,6 +15,7 @@
     "@chakra-ui/react": "^1.8.8",
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
+    "@esbuild-plugins/node-globals-polyfill": "^0.1.1",
```

Comment:
I would like to grab your attention that this lib is not well maintained and have some issues. 
You can find more here 
- https://github.com/prettier/prettier/issues/12054
- https://github.com/rollup/rollup/issues/2881


https://github.com/remorses/esbuild-plugins/blob/63b1ee1f47c7b0deb46bd71ec13d4a538e443d4d/node-modules-polyfill/src/index.ts#L39-L44


Simply put: it just paste couple of imports to every file which can lead to complex implications. 

CC: @alexkeating @jonathanprozzi

Reply:
Are you seeing a viable solution within these? Should we log this as debt and revisit?

Reply:
Actually I don't have a viable solution to suggest.
We need Keating & JP's input on that.